### PR TITLE
ecell4-4.0.0 : new formula

### PR DESCRIPTION
**Pull request by kozo2**
E-Cell4 is a multi-algorithm, multi-timescale biochemical simulation environment.
https://github.com/ecell/ecell4

### CODE REVIEW

**Comment by kozo2** [comment]
I don't know why cython call clang (not clang++) only on OSX=10.9.
https://travis-ci.org/Homebrew/homebrew-science/jobs/108005333#L4422

Do you have any idea?

File: ecell4.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  url "https://github.com/ecell/ecell4/archive/4.0.0.tar.gz"
+  sha256 "2cd3c82bc6e9666361f4d3bafa7272548ba2b52c179ad27a0fe35daf507bfd80"
+
+  head "https://github.com/ecell/ecell4.git"
+  option "with-python3", "Build python3 bindings"
+
+  depends_on "cmake" => :build
+  depends_on "gsl" => :build
+  depends_on "homebrew/versions/boost155" => :build
+  depends_on "homebrew/science/hdf5"
```
Comment (dpo) [inline ecell4.rb]: There's no need to fully qualify dependencies that are in this tap.

**Comment by dpo** [comment]
Normally, Python packages go in Homebrew/Python. But it seems that [this is available from `pip`](https://github.com/ecell/ecell4#mac), so I'm guessing it won't be acceptable.
@DomT4 ?

**Comment by DomT4** [comment]
The general rule is that libraries aren't accepted in Homebrew, although `homebrew/python` may well make exceptions there, but applications (i.e. anything that generates an executable/binary/etc) is welcome.

**Comment by kozo2** [comment]
OK, thank you for your comments.
Exactly this is available from pip.
I close this issue here.

**Pull request closed without merging**